modules/manage/partials/azure-blob-limitations.adoc

```diff
@@ -1 +1 @@
-CAUTION: Redpanda requires a container in a dedicated storage account, and doesn't support storage accounts that have https://learn.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-namespace[hierarchical namespaces^] enabled.
+NOTE: Starting in release 23.2.8, Redpanda supports storage accounts that have https://learn.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-namespace[hierarchical namespaces^] enabled. For hierarchical namespaces created with a custom endpoint, set `cloud_storage_azure_adls_endpoint` and `cloud_storage_azure_adls_port`. If you haven't configured custom endpoints in Azure, there's no need to edit these properties.
```

Comment:
@joejulian @alejandroEsc @RafalKorepta: Could one of you please confirm that we should also include this note in the Tiered Storage page for Kubernetes?

Reply:
Yes, please add this section with the pointer to change the helm chart values via 
command line flag
```
--set storage.tieredConfig.cloud_storage_azure_adls_endpoint="PLACEHOLDER" --set storage.tieredConfig.cloud_storage_azure_adls_port="PLACEHOLDER"
```

or via file
```
storage:
  tieredConfig:
    cloud_storage_azure_adls_endpoint: PLACEHOLDER
    cloud_storage_azure_adls_port: PLACEHOLDER
```

Reply:
Because this is note (similar to Linux TS page) with examples of setting properties in Helm just below it, I haven't included the actual examples. See https://deploy-preview-6--redpanda-docs-preview.netlify.app/current/manage/kubernetes/tiered-storage/?tab=tabs-9-helm#microsoft-absadls. @RafalKorepta, @JakeSCahill: let me know if it's important to include the actual examples and turn this note into a separate section.